modules/core/arrow-core-data/src/test/kotlin/arrow/core/TryTest.kt

```diff
@@ -1,5 +1,6 @@
 package arrow.core
 
+import arrow.core.extensions.*
```

Comment:
I think ktlint will fail because of this, you can modify your IDEA settings to not put on demand imports in any case